keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -97,6 +103,15 @@ func NewEdgeJoin() *cobra.Command {
 
 			return nil
 		},
+		PostRunE: func(cmd *cobra.Command, args []string) error {
+			if joinOptions.PostRun != "" {
+				step.Printf("Execute the prescript before joining the node")
```

Comment:
```suggestion
				step.Printf("Execute the postscript after joining the node")
```

Reply:
Thank you for your suggestion. I have modified it.